styles/style.css

```diff
@@ -96,3 +96,153 @@ body {
     margin-top: 40px;
 }
 
+/* Styles for main section*/
+.main-container {
+    min-width: 1440px;
+    min-height: 100vh;
+    padding: 51px 104px 120px;
+    margin: 0 auto;
+    background: url(../img/project_images/mainpage-image.png) no-repeat center center;
+}
+
+.main-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.main-header-logo {
+    width: 205px;
+    height: 40px;
+    display: block;
+    object-fit: contain;
+}
+
+.main-header-list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    position: relative;
+}
+
+.main-header-items {
+    color: #FFFFFF;
+    font-size: 24px;
+    font-weight: 400;
+    margin-left: 24px;
+}
+
+.main-header-items:first-child::before {
+    content: "";
+    position: absolute;
+    width: 50px;
+    height: 4px;
+    top: 100%;
+    left: 8%;
+    background-color: #F5BD41;
+    margin-top: 4px;
+}
+
+.night-icon {
+    background: url(../img/project_images/night-icon.png) no-repeat center center;
+    width: 30px;
+    height: 30px;
+    margin-left: 48px;
+}
+
+.account-icon {
+    background: url(../img/project_images/account-icon.png) no-repeat center center;
+    width: 40px;
+    height: 40px;
+    margin-left: 24px;
+}
+
+.main-title {
+    max-width: 542px;
+    font-size: 50px;
+    font-weight: 500;
+    line-height: 59px;
```

Comment:
This is **a bad practice**, to use absolute values for line-height
For example if you need to change font-size for mobiles - `line-height: 24px` can look badly
Using of relative values will be better
```
line-height: 2.5;
line-height: 3em;
line-height: 150%;
```

Reply:
Understood. Actually line-height property is not necessary here. Deleted at all.

Reply:
If you delete line-height for text, even one-row text, you will change space between elements, it will not be according to design, it can be hard to notice by eyes, you can use "perfect pixel" app to see the diffrence